yt_dlp/extractor/viki.py

```diff
@@ -101,9 +94,10 @@ def _check_errors(self, data):
                 if reason == 'geo':
                     self.raise_geo_restricted(msg=message)
                 elif reason == 'paywall':
+                    if try_get(data, lambda x: x['paywallable']['tvod']):
+                        self._raise_error('This video is for rent only or TVOD (Transactional Video On demand)')
```

Comment:
This will show up if they don't have purchase yet the movie. 
But for series/episode there's no for rent videos yet